README.md

```diff
@@ -1,193 +1,28 @@
 # Schema Standardization
 
-This documentation describes and explains the ReproNim schema specification.
+To see the documentation concerning the ReproNim schema specification [click]( ??? ).
 
-## 0.0: Contents
 
-- [1.0: Introduction](#10-introduction)
-- [2.0: Need for Standardizing assessments](#20-need-for-standardizing-assessments)
-- [3.0: Advantages of current representation](#30-advantages-of-current-representation)
-- [4.0: Schema](#40-schema)
-- [5.0: Contribute - how to create activity, protocol?](#50-how-can-i-create-a-new-activity-and-protocol)
-- [6.0: Test schema and collect data](#60-view-schema-and-collect-data)
-- [7.0: Why linked data?]()
-- [8.0: How these activities are licensed?]()
-- [9.0: Which tools will/are supporting this standard?]()
 
-## 1.0: Introduction
-Cognitive and clinical assessments are used throughout neuroscience, but little consistency exists in assessment data acquisition or response representation across studies. Harmonizing data after acquisition is resource intensive. Currently, the NIMH Data Archive (NDA) enforces harmonization during data submission. This approach can create a mismatch between collected and submitted data. Reverse engineering NDA data dictionaries to their original assessments using a tool like Brainverse can be tedious. To enforce consistency at the data acquisition stage, we created a standard schema and a set of reusable common assessments. The schema extends and modifies the CEDAR metadata representation. Using JSON-LD, we represent Items (elements of individual assessments) or Scores, Activities (individual assessments), and Protocols (collections of activities performed by a participant). An implementation of the schema  can specify scoring logic, branching logic, and user interface rendering options. The schema allows internationalization (multiple languages), is implementation agnostic, and tracks variations in assessments (e.g., PHQ-9, PHQ-8). This open and accessible schema library with appropriate conversion (e.g., to RedCap) and data collection tools (e.g., [MindLogger](https://mindlogger.org/), LORIS, RedCap) enables more consistent acquisition across projects, with results being harmonized by design.
+## Serving the doc locally
```

Comment:
Yes I guess that you might want to check that my instruction on how to serve the doc make sense and that anyone could do this?